learning/katas/java/Common Transforms/Aggregation/Count/src/org/apache/beam/learning/katas/commontransforms/aggregation/count/Task.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.beam.learning.katas.commontransforms.aggregation.count;
+package org.apache.beam.learning.katas.commontransforms.aggregation.count; import org.checkerframework.checker.nullness.qual.Nullable;
```

Comment:
This line is caused by my choice of the easiest search/replace solution. I expected spotless to fix it. It did not. I will file a bug against spotless. I will then fix it by adding a newline and re-running spotless.

Reply:
`learning/` is not built by Beam module plugins, so they were not applied.